### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by sntgopl**
-Created a new branch for form.
-Added HTML form structure.
-Added CSS form structure.
-Added Footer icons in HTML.
-Added Footer icons structure in CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -136,6 +136,25 @@ <h3 class="aboutmesub">Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="forms">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in </h2>
+            <form action="https://formspree.io/f/form_id" method="post">
+                <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
+                <input type="email" name="email" id="email" placeholder="Email address" required>
+                <textarea name="msg" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
+                <button type="submit">Get in touch</button>
```
Comment (Ismailco) [inline index.html:145]: - [ ] Button interactions such as hover, and active (when it's pressed), are styled as shown in the Figma template, kindly implement this so the buttons have some interactivity 

Screenshot_20220705_192332

File: index.html [hunk 2]
```diff
@@ -136,6 +136,25 @@ <h3 class="aboutmesub">Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="forms">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in </h2>
+            <form action="https://formspree.io/f/form_id" method="post">
```
Comment (Ismailco) [inline index.html]: - [ ] Your not using a valid form spree id, kindly sign up to the [formspree service](https://formspree.io/html) and create an id to use here so you can receive the emails sent from your portfolio

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @sntgopl, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding the .DS_Store to your .gitignore file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @sntgopl, 

## Status: APPROVED 🥳 🥳

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Great job with your Design 👏 
- Pull request is descriptive 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**